.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: elixir
+elixir:
+  - 1.4.1
+addons:
+  postgresql: '9.4'
```

Comment:
does Travis-CI _not_ have `v9.6.2` ? 🤔 
(_not that it matters for this PR, just **curious**_)